spec/lib/topics_filter_spec.rb

```diff
@@ -976,7 +976,101 @@
         post.topic
       end
 
-      include_examples("filtering for topics by range", "likes-op")
+      include_examples("filtering for topics by counts", "likes-op")
+    end
+
+    shared_examples "filtering for topics by date column" do |filter, column|
+      fab!(:topic) { Fabricate(:topic, column => Time.zone.local(2022, 1, 1)) }
+      fab!(:topic2) { Fabricate(:topic, column => Time.zone.local(2023, 5, 12)) }
+
+      describe "when query string is `#{filter}-after:invalid-date-test`" do
+        it "should ignore the filter" do
+          expect(
+            TopicsFilter
+              .new(guardian: Guardian.new)
+              .filter_from_query_string("#{filter}-after:invalid-date-test")
+              .pluck(:id),
+          ).to contain_exactly(topic.id, topic2.id)
+        end
+      end
+
+      describe "when query string is `#{filter}-after:2022-01-01`" do
+        it "should only return topics with bumped at after 2022-01-01" do
```

Comment:
This is a shared test case, but the description here is hardcoded to say "... bumped at ..." in all contexts which isn't accurate because we use this shared example to test sorting by `created_at` and `last_posted_at`.

I think we need to add a third argument to the shared example for the description.

(the 5 test cases below have the same problem — they all say "bumped at")

Reply:
Good catch! I've fixed the tests.